lib/versions.js

```diff
@@ -33,3 +33,34 @@ module.exports.getVersionInfo = (lastRelease, template) => {
     })
   }
 }
+
+module.exports.incrementVersionBasedOnLabels = (lastRelease, pullRequests) => {
+  const lastVersion = this.getVersionInfo(lastRelease)
+  const winningLabel = getWinningLabelFrom(pullRequests)
+  switch (winningLabel) {
+    case 'MINOR':
```

Comment:
Not sure hard coding these is the best idea 😕 
Much better to allow a list of labels to bump.

I could foresee someone wanting `feature` or `breaking` labels to allow them bumping.

Reply:
Not sure what you mean by 'list of labels to bump', sorry -- could you provide an example? Do you mean that users should be able to specify their own set of labels -> actions alongside some sensible defaults?

Reply:
This is just a given example of labels that could be declared to auto bump

I could foresee wanting to enable autoRelease on a org level, see https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml
```yaml
autoRelease:
  enabled: false
  major:
    - major
    - breaking
  minor:
    - minor
    - feature
    - sparkles
  patch:
    - patch
    - bug
    - bugfix
```

Then inside my repo https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/.github/release-drafter.yml

I could have

```yaml
_extends: .github
name-template: v$NEXT_MINOR_VERSION 🌈
tag-template: configuration-as-code-$NEXT_MINOR_VERSION
autoRelease:
  enabled: true
```

so because of probot-config the autoRelease will be deep merged and look like

```yaml
autoRelease:
  enabled: true
  major:
    - major
    - breaking
  minor:
    - minor
    - feature
    - sparkles
  patch:
    - patch
    - bug
    - bugfix
```

Reply:
Cool, I see - great idea! Will try and get on this and your other comment soon :)

Reply:
This is now covered by the calling code -- depending on the list of labels given by a user or defaults, it maps it to one of these three options.